lib/commands/help.js

```diff
@@ -13,7 +13,7 @@ const globify = pattern => pattern.split('\\').join('/')
 // We don't currently compress our man pages but if we ever did this would
 // seamlessly continue supporting it
 const manNumberRegex = /\.(\d+)(\.[^/\\]*)?$/
-// hardcoded names for mansections
+// hardcoded names for manual sections
```

Comment:
```suggestion
// hardcoded names for man sections
```

"man" is a technical term here and is appropriate to use in this case.  The variable is `manSectionNames` for this reason.